andela/activo-web
# #157506226 Fix navigation from the centers page to spaces page

**What does this PR do?**

This PR fixes the navigation bug between the `CentersPage` and `SpacesPage`

**Description of Task to be completed?**

Fix the navigation bug while navigating from the `CentersPage` to the `SpacesPage`

**How should this be manually tested?**

Navigate to `settings > centers` click on a center and you should be navigated to the SpacesPage for that center

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#160088367 Fix navigation to Spaces from centers](https://www.pivotaltracker.com/story/show/160088367)

**Screenshots (if appropriate)**

xxx

**Questions:**

xxx
